src/benchmark/operators/table_scan_sorted_benchmark.cpp

```diff
@@ -105,35 +118,55 @@ std::shared_ptr<TableWrapper> create_table(const DataType data_type, const int t
 
 void BM_TableScanSorted(
     benchmark::State& state, const int table_size, const double selectivity, const EncodingType encoding_type,
-    const std::string& mode,
+    const std::string& mode, const bool is_between_scan,
```

Comment:
const

Reply:
What do you mean by that?

Reply:
I think I wanted to say that you missed using a `const` somewhere and missed the exact line.